scripts/development/k8s/elasticsearch-kibana/elasticsearch.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: elasticsearch
+  namespace: dracon
+spec:
+  serviceName: "elasticsearch"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: elasticsearch
+  template:
+    metadata:
+      labels:
+        app: elasticsearch
+    spec:
+      containers:
+        - name: elasticsearch
+          image: docker.elastic.co/elasticsearch/elasticsearch:7.6.0
```

Comment:
https://github.com/thought-machine/pleasings/blob/master/docker/docker.build_defs doesn't have `docker_mirror`, which we'd need to support this.

I'm also doing the replacement based on the FQN as it is now instead of `//<plz-target>` in my next PR as it provides a default for people who aren't using plz elsewhere (and can use kustomize to pin the tags reasonably) e.g. `index.docker.io/thoughtmachine/dracon-producer-gosec:latest` will become `index.docker.io/thoughtmachine/dracon-producer-gosec:22dbaf39b7ad5323e6b932c75e7c474bc37d36c7838a70fb69a5c9be7540edaf`